src/vtcomposite.cpp

```diff
@@ -91,9 +91,9 @@ struct BatonType
     bool compress = false;
 };
 
-struct InternationalizeBatonType
+struct localizeBatonType
```

Comment:
convention has data structures in `PascalCase`
```suggestion
struct LocalizeBatonType
```

Reply:
woops, my find + replace all wasn't perfect!